src/lib.rs

```diff
@@ -28,6 +28,7 @@ pub mod transports;
 pub mod types;
 
 pub use api::{Web3Main as Web3, ErasedWeb3};
+pub use helpers::CallResult;
```

Comment:
CallResult also isn't the best name...it should be called "LazyDecode" or something like that